test/test_transforms.py

```diff
@@ -230,10 +230,10 @@ def _test_librosa_consistency_helper(n_fft, hop_length, power, n_mels, n_mfcc, s
             librosa_mel = librosa.feature.melspectrogram(y=sound_librosa, sr=sample_rate,
                                                          n_fft=n_fft, hop_length=hop_length, n_mels=n_mels,
                                                          htk=True, norm=None)
-            librosa_mel_tensor = torch.from_numpy(librosa_mel)
             torch_mel = melspect_transform(sound).squeeze().cpu().t()
 
-            self.assertTrue(torch.allclose(torch_mel.type(librosa_mel_tensor.dtype), librosa_mel_tensor, atol=5e-3))
+            # lower tolerance, think it's double vs. float
+            self.assertTrue(torch.allclose(torch_mel.type(torch.double), torch.from_numpy(librosa_mel), atol=5e-3))
```

Comment:
That error is essentially 1e-2. What is the relative scale we are taking about here? rtol=1e-4?

Reply:
A separate PR, if needed, can be made to fix this as it is unrelated to istft. There is a PR to fix the CI but that is strictly for fixing the build